subiquity/server/controllers/filesystem.py

```diff
@@ -294,6 +294,18 @@ def load_autoinstall_data(self, data):
         # log.debug("self.ai_data = %s", data)
         self.ai_data = data
 
+    # The identity and user-data section are optional if we are only installing
+    # the reset partition, however the identity controller needs to know this
+    # before the filesystem controller naturally sets this. So this is a
+    # temporary function to inspect the outcome.
+    # See: https://github.com/canonical/subiquity/pull/1965
+    def _reset_parition_only(self):
```

Comment:
Ah man I really can't type partition lately. Thanks. 

>  Probably should not have the leading underscore either, as the intention is that it can be used by other controllers.

Yeah good point. Renamed to `is_reset_partition_only` to not conflict with the instance variable `reset_partition_only`